.github/workflows/go.yml

```diff
@@ -7,16 +7,25 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        go: ["1.23.x", "1.22.x", "1.21.x", "1.20.x", "1.19.x", "1.18.x"]
+        go: ["1.19.x","1.18.x","1.7.x"] # code works with 1.3.x but we need 1.7.x for Benchmark.Run
 
     steps:
       - name: Setup Go ${{ matrix.go }}
-        uses: actions/setup-go@v5
```

Comment:
GOPATH is deprecated.
https://go.dev/wiki/GOPATH#:~:text=See%20the%20go%20command%20documentation,mode%2C%20which%20is%20now%20deprecated.

Reply:
See above.  I reverted to your file as that change was unintended.